_chapters/chapter_01.md

```diff
@@ -72,13 +80,21 @@ layout: rails_tutorial
 {% endhighlight %}
 {% endsteps %}
 
+{% aside %}
+  The command "bundle" might sound familiar to you. If you did the install instructions for this tutorial, you probably ran `$gem install bundler`. 
+
+  [Bundler](http://bundler.io/) is a program that helps you manage all of the other little programs that help Rails run and all of the commands related to that program start with `$bundle`. 
+
+  You don't need to worry about this much for the tutorial but if you're interested in it, you can learn more about them on their website. 
+{% endaside %}
+
 {% steps %}
 {% list %}
-  1.  Let's take a look the files that got created.
+  1.  Let's take a look the files that got created when you ran `$rails new`.
 
-  1.  Run `ls -l` to list out all the top level files and directories `rails new` generated. There is lot going on, but a few things stand out.
+  1.  Run `ls -l` to list out all the top level files and directories that you created. There's lot going on, but a few things stand out.
```

Comment:
ya - i just took it out because it's redundant. the point above it sets up that we're talking about that ... and it sounds weird to have the phrase rails new over and over.